wcomponents-core/src/main/java/com/github/bordertech/wcomponents/render/webxml/WTableRenderer.java

```diff
@@ -301,11 +301,13 @@ private void paintTableActions(final WTable table, final WebXmlRenderContext ren
 						int minRows = constraint.getMinSelectedRowCount();
 						int maxRows = constraint.getMaxSelectedRowCount();
 						String message = constraint.getMessage();
+						int selectedOnOther = table.getSelectedRows().size();
 						String type = constraint.isError() ? "error" : "warning";
 
 						xml.appendTagOpen("ui:condition");
 						xml.appendOptionalAttribute("minSelectedRows", minRows > 0, minRows);
 						xml.appendOptionalAttribute("maxSelectedRows", maxRows > 0, maxRows);
+						xml.appendOptionalAttribute("selectedOnOther", selectedOnOther);
```

Comment:
What does `appendOptionalAttribute` do? 
Will it ever behave any differently to `appendAttribute` in this scenario? 
This line needs to change, see if you can work out how.

Reply:
Depending on how this attribute pans out the schema may need an update to reflect either that the attribute is always present or to declare the assumed value if it is an optional attribute.

Reply:
I was tossing up between appendOptionalAttribute or appendAttribute, and chose appendOptionalAttribute as, from my understanding, it would only add the XML attribute when it is not null, and that it wouldn't show when it is. I figured we only really need to know whether there is items selected on other pages when there is a constraint, and not need to know all the time, therefore making it optional.

I've only just realised now though that a) I forgot to add a flag to show only when there is actually a constraint and b) selectedOnOther is never null (will be an empty set at least, but not null) and so it is always showing anyway, so it might as well be appendAttribute. Depending on what solution I find, this attribute may not even be needed in the end.